client/me/help/help-contact-form/index.jsx

```diff
@@ -395,6 +442,51 @@ export class HelpContactForm extends React.PureComponent {
 			{ value: 'panicked', label: translate( 'Panicked' ) },
 		];
 
+		const siteData = this.state.siteData && this.state.siteData.site;
+		const errorData = this.state.siteData && this.state.siteData.errorCode;
+
+		let noticeMessage;
+		let actionLink;
+		let actionMessage;
+
+		// "Unauthorized" means it's still a WP.com site - just a private one.
+		const isWpComConnectedSite =
+			( siteData && siteData.ID ) ||
+			( errorData && errorData === 'unauthorized' && this.state.userDeclaredUrl );
+
+		const isNonWpComHostedSite =
+			errorData && errorData === 'unknown_blog' && this.state.userDeclaredUrl;
+
+		if ( isWpComConnectedSite ) {
+			// The site is linked to WordPress.com but not appearing for the user, so they've probably lost access to the account which owns it.
+			noticeMessage = translate(
+				"%(siteName)s is linked to another WordPress.com account. If you're trying to access it, please follow our Account Recovery procedure.",
+				{
+					args: {
+						siteName:
+							siteData && siteData.name
+								? siteData.name
+								: translate( 'This site', {
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **45** times:
`translate( 'This Site' )` *ES Score: 6*